fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -41,7 +39,7 @@ public class ImpactAreaStageDamage : Validation, IReportMessage
         public event MessageReportedEventHandler MessageReport;
         #endregion
         #region Constructor
-        public ImpactAreaStageDamage(int impactAreaID, Inventory inventory, List<OccupancyType> occupancyTypes, HydraulicDataset hydraulicDataset, ConvergenceCriteria convergence, ContinuousDistribution analyticalFlowFrequency = null, GraphicalUncertainPairedData graphicalFrequency = null,
```

Comment:
good catch - looks like occ types are a property of an inventory. we'll need to update the scenario class to reflect this change if not done already